app/models/solidus_subscriptions/checkout.rb

```diff
@@ -118,7 +117,8 @@ def subscription
     end
 
     def ship_address
-      subscription.shipping_address || user.ship_address
+      # subscription.shipping_address || user.ship_address
+      subscription.shipping_address || user.ship_address || user.bill_address
```

Comment:
This seems like it should probably go something like:

```
subscription.shipping_address || subscription.order.shipping_address || user.ship_address || user.bill_address
```

That direct association between subscription and order does not exist (I don't think) but it should be simple to derive.